onnxscript/rewriter/onnxruntime/instance_to_group_normalization.py

```diff
@@ -93,7 +98,16 @@ def check_if_simulated_instance_norm_is_used(
     Returns:
         bool: True if the simulated instance normalization is used, False otherwise.
     """
-    return _check_if_simulated_instance_norm_is_used_impl(**match_bindings)
+    return _check_if_simulated_instance_norm_is_used_impl(
```

Comment:
nit: I think we can just merge these two functions into one, now that the caller is doing the `**`